samples/sampler/stories/qa/ProgressBar.js

```diff
@@ -1,23 +1,35 @@
-// import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
-// import {mergeComponentMetadata} from '@enact/storybook-utils';
-// import {storiesOf} from '@storybook/react';
-//
-// import ProgressBar, {ProgressBarBase} from '@enact/moonstone/ProgressBar';
-//
-// const Config = mergeComponentMetadata('ProgressBar', ProgressBarBase, ProgressBar);
-//
-// storiesOf('ProgressBar', module)
-// 	.add(
-// 		'The basic ProgressBar',
-// 		() => (
-// 			<ProgressBar
-// 				backgroundProgress={number('backgroundProgress', Config, 0.5, {range: true, min: 0, max: 1, step: 0.01})}
-// 				tooltip={boolean('tooltip', Config, false)}
-// 				highlighted={boolean('highlighted', Config, false)}
-// 				progress={number('progress', Config, 0.4, {range: true, min: 0, max: 1, step: 0.01})}
-// 				orientation={select('orientation', ['horizontal', 'vertical'], Config, 'horizontal')}
-// 				disabled={boolean('disabled', Config, false)}
-// 			/>
-// 		),
-// 		{propTables: [Config]}
-// 	);
+import ProgressBar, {ProgressBarBase} from '@enact/moonstone/ProgressBar';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {boolean, number, select} from '@enact/storybook-utils/addons/controls';
+
+const Config = mergeComponentMetadata('ProgressBar', ProgressBarBase, ProgressBar);
+
+export default {
+	title: 'Moonstone/ProgressBar',
+	component: 'ProgressBar'
+};
+
+export const BasicProgressBar = (args) => {
+	return (
+		<ProgressBar
+			backgroundProgress={args['backgroundProgress']}
+			disabled={args['disabled']}
+			highlighted={args['highlighted']}
+			orientation={args['orientation']}
+			progress={args['progress']}
+			tooltip={args['tooltip']}
+		/>
+	);
+};
+
+number('backgroundProgress', BasicProgressBar, Config, 0.5, {range: true, min: 0, max: 1, step: 0.01});
```

Comment:
Even though they were numbers before please replace both backgroundProgress and progress with range as number does not seem to work properly with step.